pehlaschool/proj.android-studio/app/src/org/cocos2dx/cpp/DownloadExpansionFile.java

```diff
@@ -45,13 +45,13 @@
     public static final XAPKFile[] xAPKS = {
             new XAPKFile(
                     true, // true signifies a main file
-                    2, // the version of the APK that the file was uploaded
-                    1380598140L // the length of the file in bytes
+                    4, // the version of the APK that the file was uploaded
```

Comment:
Should be 5 for next release

Reply:
We will use patch file for that .. since the English file is not ready yet we don't know the correct size of the OBB file yet